src/main/java/uk/gov/justice/digital/hmpps/cmd/api/repository/PrisonRepository.kt

```diff
@@ -7,4 +7,5 @@ import java.util.*
 
 @Repository
 interface PrisonRepository : CrudRepository<Prison, UUID> {
+    override fun findAll(): Collection<Prison>
```

Comment:
For the sake of logging how many we return we should use Collection not Iterable because Iterable doesn't have a size() method.